SEED-platform/seed
# Update the backup script to delete files older than 30 days

#### Any background context you want to provide?

The backup script previously deleted database and media file backups older than 45 days.  This is eating up more disk space on the servers than we'd like.

#### What's this PR do?

This alters the backup script to delete files older than 30 days.

#### How should this be manually tested?

Run the script and verify that files older than 30 days in ~/seed-backups are indeed removed.

#### What are the relevant tickets?

N/A

#### Screenshots (if appropriate)
